src/debian/auditd/ioperm.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
```

Comment:
I think these files are only used for testing and should rather be moved to the cukinia-tests repository.
We don't want testing executables to be deployed on a production system for cybersecurity.

Reply:
I moved them in Cukinia repository, see the commit: https://github.com/seapath/cukinia-tests/pull/30/commits/ab895ce098ebfecdd8787858b7bdd189303797ea